src/components/FindABook/FindABook.jsx

```diff
@@ -0,0 +1,17 @@
+import '../BookCard/BookCard.css';
+import BookCard from '/src/components/BookCard/BookCard.jsx';
+import booksData from '/src/data/booksData.js';
+
+
+export default function FindABook({
+	
```

Comment:
This doesn't need to be an empty line if there are no props. You can also get rid of the `{}` since we are not de-structuring anything.